src/node.rs

```diff
@@ -144,6 +153,9 @@ pub enum NodeName {
 
     /// Name separated by colons, e.g. `<div on:click={foo} />`
     Colon(Punctuated<Ident, Colon>),
+
+    /// Arbitrary rust code in braced `{}` blocks
+    Block(Expr),
```

Comment:
Just realized, is there a specific reason that this is `Expr` and not `ExprBlock`?